### DISCUSSION TITLE
Creating a Mapbox token correctly

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1368] [Q&A] [answered]
We have just set up the Mapbox account and creating the token.

Do we leave the options pretty much as-is and just add the domains (production and staging) on which it will be used? Or do we have to configure this differently?

There are a lot of checkboxes!

Thank you so much

![mapbox token](https://github.com/code4recovery/12-step-meeting-list/assets/159723102/e24dd9ac-a886-4f8c-a56d-0725d6ef2333)

**Comment by willinrecovery** [comment]
Looks like we didn't get it quite right as the map is not showing correctly.

It's looking great though so far, don't you think?

<img width="958" alt="Monosnap CECDA Staging 2024-02-23 12-36-35" src="https://github.com/code4recovery/12-step-meeting-list/assets/159723102/a8c76be7-985b-4645-95d3-1dbd9c05d830">

**Reply by gkovats** [reply]
What url do you have @willinrecovery ? Maybe the response from mapbox gives some clue?

**Reply by willinrecovery** [reply]
We first tried entering the URLs for the production and staging server.

Then, when that didn't work, we tried entering the URL for the meetings page and even the individual meeting detail.

<img width="1015" alt="Monosnap Account | Mapbox 2024-02-23 16-23-48" src="https://github.com/code4recovery/12-step-meeting-list/assets/159723102/cbd250be-e9cd-4499-bcea-cf782eee07e4">

It is still not working.

**Comment by joshreisner** [comment, marked as answer]
@willinrecovery check to make sure the key is entered in the Mapbox field on the Meetings > Settings page (and not the Google field)

**Reply by willinrecovery** [reply]
😳 that was the problem. Thank you for your patience. It is working now!
